configs/gpu/gpu-function.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: hello-gpu
+spec:
+  template:
+    spec:
+      containers:
+        - image: ghcr.io/vhive-serverless/hello-gpu:latest
```

Comment:
can you please add the source code and the dockerfile to `function-images/`? or leave the link to the source code in a comment